src/app/launcher/mission-runtime-step/mission-runtime-step.component.html

```diff
@@ -24,9 +24,10 @@ <h2 class="card-pf-title">
             </div>
             <div class="card-pf-body">
               <div class="list-group list-view-pf" *ngFor="let runtime of runtimes; let i = index">
-                <div class="list-group-item list-view-pf-stacked list-view-pf-top-align">
+                <div class="list-group-item list-view-pf-stacked list-view-pf-top-align" [ngClass]="{'disabled': runtime.disabled === true}">
```

Comment:
FYI, the disabled state should be typed via the runtime.model.ts file. The REST APIs will populate this Runtime model -- will the API also provide the disabled state?

Reply:
@dlabrecq - we need to set disabled based on the support runtime or mission in the opposite column (so if a runtime is selected and the mission in the other column isn't in the supported list, it should be disabled)